app/controllers/bgeigie_imports_controller.rb

```diff
@@ -45,6 +45,11 @@ def reject
   def send_email
     @bgeigie_import = BgeigieImport.find(params[:id])
     @bgeigie_import.send_email(params[:email_body])
+    @bgeigie_import.uploader_contact_histories.create!(
+      administrator: current_user, previous_status: @bgeigie_import.status
+    )
+    # use update_columns to avoid validations of presence of cities and credits
```

Comment:
Not for this PR, but I think these validations should get removed anyway and instead moved into controller-level check during the "submit for approval" process.